modules/groundside_scripts.py

```diff
@@ -0,0 +1,162 @@
+"""
+Groundside scripts to receive GPS location messages
+"""
+
+import struct
+import sys
+from pymavlink import mavutil
+
+# connection params
+CONNECTION_ADDRESS = "tcp:127.0.0.1:14550"
+TIMEOUT = 5.0
+DELAY_TIME = 1.0
+
+# file path to read from
+# 1. start up connection on mission planner
+# 2. mavlink and establish write tcp connection
+# 3. config menu -> MAVFtp -> simulation drone file paths
+FILE_PATH = b"/@ROMFS/locations.txt"
+SEQ_NUM = 0
+
+
+def send_ftp_command(
+    connection: mavutil.mavlink_connection,
+    seq_num: int,
+    opcode: int,
+    req_opcode: int,
+    session: int,
+    offset: int,
+    size: int,
+    payload: bytes,
+) -> None:
+    """
+    Send an FTP command to the vehicle.
+
+    Args:
+        connection (mavutil.mavlink_connection): MAVLink connection object.
+        opcode (int): FTP command opcode.
+        req_opcode (int): Requested opcode.
```

Comment:
Since you will never be sending a ACK or NAK message, this probably shouldn't be a parameter. You can always set it to 0 or whatever